src/main/java/teammates/ui/controller/InstructorStudentCommentAddAction.java

```diff
@@ -61,12 +62,14 @@ protected ActionResult execute()  throws EntityDoesNotExistException {
             logic.putDocument(createdComment);
             
             statusToUser.add(Const.StatusMessages.COMMENT_ADDED);
-            statusToAdmin = "Created Comment for Student:<span class=\"bold\">(" +
-                    comment.recipients + ")</span> for Course <span class=\"bold\">[" +
-                    comment.courseId + "]</span><br>" +
-                    "<span class=\"bold\">Comment:</span> " + comment.commentText;
-        } catch (EntityAlreadyExistsException e) {  // this exception should not be thrown normally unless GAE creates duplicate commentId
-            Assumption.fail("Creating a duplicate comment should not be possible as comments should have different timestamp\n");
+            statusToAdmin = "Created Comment for Student:<span class=\"bold\">("
+                    + comment.recipients + ")</span> for Course <span class=\"bold\">["
+                    + comment.courseId + "]</span><br>"
+                    + "<span class=\"bold\">Comment:</span> " + comment.commentText;
```

Comment:
Indent these three lines more so that it appears on the right side of the `=` sign.